src/Health/HealthDAO.ts

```diff
@@ -33,15 +33,15 @@ export interface IPFSInfo {
   readonly ipfsIsConnected: boolean
 }
 
-type updateBlockchainInfo = (x: BlockchainInfo) => Promise<UpdateWriteOpResult>
+type updateBlockchainInfo = (x: BlockchainInfo) => void
```

Comment:
should be Promise<void> I think? wonder why ts is complaining

Reply:
try, Promise<void>

Reply:
yup that is what I meant, man... my brain today.

Reply:
Right, TS is complaining because we're trying to tell it "this returns a Promise<void>" but it knows better. Maybe a type cast? `return ... as Promise<void>`